flumine/flumine.py

```diff
@@ -71,14 +63,23 @@ def _add_default_workers(self):
                 start_delay=10,  # wait for streams to populate
             )
         )
-        self.add_worker(
-            worker.BackgroundWorker(
-                self,
-                function=worker.poll_market_closure,
-                interval=60,
-                start_delay=10,  # wait for login
+        if self.client.market_recording_mode != True:
```

Comment:
Can you change to be more pythonic

```
if self.client.market_recording_mode:
```

Reply:
This breaks the tests (test__add_default_workers), because flumine.client.market_recording_mode is an Object

Reply:
Sorry I mean

```python
if self.client.market_recording_mode is False:
```

And yeah the text will need to be fixed with

```python
self.flumine.client.market_recording_mode = False
```

Reply:
Great, thanks